libs/sdk-react-native/android/src/main/java/com/breezsdk/BreezSDKMapper.kt

```diff
@@ -156,16 +156,23 @@ fun pushToArray(array: WritableArray, value: Any?) {
     when (value) {
         null -> array.pushNull()
         is Boolean -> array.pushBoolean(value)
+        is BuyBitcoinRequest -> array.pushMap(readableMapOf(value))
+        is BuyBitcoinResponse -> array.pushMap(readableMapOf(value))
         is Double -> array.pushDouble(value)
         is FiatCurrency -> array.pushMap(readableMapOf(value))
         is Int -> array.pushInt(value)
         is LocaleOverrides -> array.pushMap(readableMapOf(value))
         is LocalizedName -> array.pushMap(readableMapOf(value))
         is LspInformation -> array.pushMap(readableMapOf(value))
+        is OpeningFeeParams -> array.pushMap(readableMapOf(value))
```

Comment:
This is the only one needed here as it's used in OpeningFeeParamsMenu values array. The others could be removed.

Reply:
Done in 3b9ef403befd5079c2e761d6dacd0f0beda0e125